app/components/ChallengesMapView.js

```diff
@@ -71,17 +70,17 @@ export default class ChallengesMapView extends Component {
 				<View style={{flex:1}}>
 					<MapView
 						style={styles.map}
-						loadingEnabled={true}
-						showUserLocation={true}
+						loadingEnabled={false}
```

Comment:
@jwt1143 I've talked to yoojeong. It looks like the fix is to add
```javascript
{
  enableHighAccuracy: true,
  timeout: 200000,
  maximumAge: 1000,
  distanceFilter: 10
}
```
as a third parameter to the `getCurrentPosition` function in the MainMapScene.js. However, that being said, yoojeong said it still took 2-3 minutes for the map to load, which is insane! I've been waiting longer than that and it still isn't loading for me.

Now, it may not technically be the correct thing to do, but if I set `loadingEnabled` to `false` the map loads in like .5 seconds. So...